scripts/run-component-tests.js

```diff
@@ -44,9 +44,9 @@ shelljs.set('-e');
 shelljs.cd(projectDir);
 
 // Extracts the supported command line options.
-const {_: components, local, firefox, watch} = minimist(args, {
+const {_: components, local, firefox, watch, config} = minimist(args, {
```

Comment:
I was considering between calling it `config` or something like `--view-engine` since `--config` isn't super obvious to somebody that isn't familiar with the setup. I went with `config`, because it's consistent, but I wouldn't mind renaming it.

Reply:
I don't mind too much either. I personally would prefer `--view-engine` since we try to make the workflow more intuitive (than what Bazel allows us). I think `--config` might be more confusing than helpful. Ivy is the default, and the opt-in view engine flag could change that. Up to you though 😄